modules/assisted-installer-assisted-installer-prerequisites.adoc

```diff
@@ -0,0 +1,60 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='assisted-installer-prerequisites_{context}']
+= Assisted Installer prerequisites
+
+The {ai-full} validates the following prerequisites to ensure successful installation.
+
+== Hardware
+
+For control plane nodes or the {sno} node, nodes must have at least the following resources.
+
+* 8 CPU cores
+* 16.00 GiB RAM
+* 100 GB storage
+* 10ms write speed or less for etcd `wal_fsync_duration_seconds`
+
+For worker nodes, each node must have at least the following resources.
+
+* 4 CPU cores
+* 16.00 GiB RAM
+* 100 GB storage
+
+== Networking
+
+The network must meet the following requirements.
+
+* A DHCP server unless using static IP addressing.
+* A base domain name. You must ensure:
+  - There is no wildcard, such as `*.<cluster_name>.<base_domain>`, or the installation will not proceed.
+  - A DNS A/AAAA record for `api.<cluster_name>.<base_domain>`.
+  - A DNS A/AAAA record with a wildcard for `*.apps.<cluster_name>.<base_domain>`.
+* Port `6443` is open for the API URL if you intend to allow users outside the firewall to access the cluster via the `oc` CLI tool.
+* Port `443` is open for the console if you intend to allow users outside the firewall to access the console.
+
+[IMPORTANT]
+====
+DNS A/AAAA record settings at top-level domain registrars can take significant time to update. Ensure the A/AAAA record DNS settings are working before installation to prevent installation delays.
+====
+
+The {product-title} cluster's network must also meet the following requirements.
+
+* Connectivity between all cluster nodes
+* Connectivity for each node to the internet
+* Access to an NTP server for time synchronization between the cluster nodes
+
+== Preflight validations
+
+Ensuring the cluster meets the prerequisites before installation is beneficial, because it eliminates complex post-installation troubleshooting, thereby saving significant amounts of time and effort. Before installing software on the nodes, the {ai-full}:
```

Comment:
This sentence should be completed, as per IBMSG: Use a colon after the introduction to a vertical list, including a procedure or substeps in a procedure. To avoid translation problems, make the introduction a complete sentence.

https://www.ibm.com/docs/en/ibm-style?topic=punctuation-colons